data/sheragi/archaeology missions.txt

```diff
@@ -339,17 +339,19 @@ mission "Sad Archie"
 		has "There Might Be Riots part 3B: done"
 	destination "Zug"
 	
-	npc disable
-		government "Test Dummy"
-		system "Ildaria"
-		personality unconstrained heroic nemesis
-		ship "Timer Ship" "Timer"
-		conversation
-			`As your ship drifts through the cloud of dust that surrounds Ildaria, it occurs to you that this must be the "cloudy star out along the Rim" that Ulrich from There Might Be Riots told you about. Remembering his story, you shut your eyes and attempt to meditate. But instead, you just fall asleep.`
-			scene "scene/eso0"
-			`You dream that you leave this star system and return to Zug, but when you land there everything is different: the cities you remember are gone, and in their place are other cities with strange architecture, populated by gargantuan dragon-like creatures.`
-			`	Time speeds up, and you watch as the dragon cities grow and spread out. You see one city buried under a massive lava flow, starships buzzing around it like a cloud of flies to carry people and goods to safety. Other cities are destroyed by war, and then abruptly the dragon-people are all gone and their cities lie vacant. Buildings crumble, and the forests reclaim the land.`
-			`	Then you wake up and find yourself back in your ship, in the Ildaria system. Weird.`
+	timer "Sad Archie Timer"
+		time 7200 3600
+		system Ildaria
+		idle
+		uncloaked
+		reset "leave system"
+		on timeup
+			conversation
+				`As your ship drifts through the cloud of dust that surrounds Ildaria, it occurs to you that this must be the "cloudy star out along the Rim" that Ulrich from There Might Be Riots told you about. Remembering his story, you shut your eyes and attempt to meditate. But instead, you just fall asleep.`
+				scene "scene/eso0"
+				`You dream that you leave this star system and return to Zug, but when you land there everything is different: the cities you remember are gone, and in their place are other cities with strange architecture, populated by gargantuan dragon-like creatures.`
+				`	Time speeds up, and you watch as the dragon cities grow and spread out. You see one city buried under a massive lava flow, starships buzzing around it like a cloud of flies to carry people and goods to safety. Other cities are destroyed by war, and then abruptly the dragon-people are all gone and their cities lie vacant. Buildings crumble, and the forests reclaim the land.`
+				`	Then you wake up and find yourself back in your ship, in the Ildaria system. Weird.`
```

Comment:
Data changes with code :raised_eyebrow:

Reply:
Is this Not Done™? I can break this out into a separate PR if that's preferable. It's here for two reasons—first, to demonstrate the usage, and second, to make sure that _as soon as_ this feature is merged, Sad Archie can make use of it and stop being a crapshoot.

Reply:
Well I dont like it, there might be different opinions though

Reply:
it's litteraly the main use case for this, makes sense we'd use it